libs/zsdl/src/sdl2.zig

```diff
@@ -1056,6 +1056,117 @@ pub const Surface = opaque {
 // Platform-specific Window Management
 //
 //--------------------------------------------------------------------------------------------------
+pub const SysWMType = enum(i32) {
+    unknown,
+    windows,
+    x11,
+    directfb,
+    cocoa,
+    uikit,
+    wayland,
+    mir, // no longer available, left for api/abi compatibility. remove in 2.1!
+    winrt,
+    android,
+    vivante,
+    os2,
+    haiku,
+    kmsdrm,
+    riscos,
+};
+
+/// Get driver-specific information about a window.
+///
+/// The caller must initialize the `info` structure's version by using
+/// `VERSION(&info.version)`, and then this function will fill in the rest
+/// of the structure with information about the given window.
+///
+/// returns true if the function is implemented and the `version` member
+/// of the `info` struct is valid, or false if the information
+/// could not be retrieved
+///
+/// ZSDL Note: Only implemented for SysWMType.windows & SysWMType.x11
+pub fn getWindowWMInfo(window: *Window, info: *SysWMInfo) bool {
+    const result: bool = SDL_GetWindowWMInfo(window, info);
+    return switch (info.subsystem) {
+        .windows, .x11 => result,
+        else => false,
+    };
```

Comment:
Why don't we just return the result of `SDL_GetWindowWMInfo` here? Why are windows and x11 special?